routes/dbapi.js

```diff
@@ -0,0 +1,60 @@
+const MongoClient = require('mongodb').MongoClient;
+const autoIncrement = require('mongodb-autoincrement');
+const dbUrl = 'mongodb://localhost:27017/guess';
+
+let _db;
+
+/**
+ * MongoDB operations interface
+ */
+function db() {
+  MongoClient.connect(dbUrl, function(err, database) {
+    if (err) return console.log(err);
+    _db = database;
+  });
+}
+
+db.prototype.insertInCollection = function(collection, payload, callback) {
+	// validate(req.body)
+	let collec = _db.collection(collection);
+	// Check if word already exists
```

Comment:
not sure if checking this inside `insertInCollection` is the right way, these methods should be pretty atomic,
and may be in final api  we can call 2 methods instead of one